_i18n/de/resources/moneropedia/encryption.md

```diff
@@ -1,34 +1,34 @@
 
-### In-depth information
+(Frei übersetzt von [Encryption](https://en.wikipedia.org/wiki/Encryption))
 
-From [Encryption](https://en.wikipedia.org/wiki/Encryption):
+### Ausführliche Informationen
 
 >
- In an encryption scheme, the intended communication information or message (referred to as *plaintext*), is encrypted using an encryption algorithm, generating ciphertext that can only be read if decrypted. For technical reasons, an encryption scheme usually uses a pseudo-random encryption key generated by an algorithm. It is in principle possible to decrypt the message without possessing the key, but, for a well-designed encryption scheme, large computational resources and skill are required. An authorized recipient can easily decrypt the message with the key provided by the originator to recipients, but not to unauthorized interceptors.
+Mit einem Verschlüsselungssystem wird die ursprüngliche Information oder Nachricht (*"Klartext"* genannt) unter Verwendung eines Verschlüsselungsalgorithmus chiffriert und somit ein "Geheimtext" generiert, welcher nur nach Entschlüsselung lesbar ist. Aus technischen Gründen verwendet ein Verschlüsselungssystem einen pseudozufälligen, durch einen Algorithmus generierten Verschlüsselungscode. Prinzipiell ist es möglich, die Nachricht zu dechiffrieren, ohne über den Code zu verfügen; bei einem gut aufgebauten Verschlüsselungssystem benötigt dies jedoch umfangreiche Rechenressourcen und Können. Ein autorisierter Empfänger kann die Nachricht ganz einfach mit dem Code entschlüsseln, welchen der Absender jeglichen Empfängern, nicht jedoch unautorisierten Abfängern, bereitgestellt hat.
 
 >
-The purpose of encryption is to ensure that only somebody who is authorized to access data (e.g. a text message or a file), will be able to read it, using the decryption key. Somebody who is not authorized can be excluded, because he or she does not have the required key, without which it is impossible to read the encrypted information.
+Der Zweck von Verschlüsselung ist es, sicherzustellen, dass nur jemand, der zum Abruf von Daten (bspw. einer Textnachricht oder einer Datei) autorisiert ist, ebendiese unter Verwendung des Entschlüsselungscodes lesen kann. Jemand, der nicht autorisiert ist, kann ausgeschlossen werden, da der- oder diejenige nicht über den benötigten Code verfügt, ohne welchen es unmöglich ist, die verschlüsselte Information zu lesen.
+
+(Frei übersetzt von [Encryption](https://en.wikipedia.org/wiki/Encryption))
 
 ### Kovri
 
-@Kovri implements various types of encryption in *at least* 4 essential capacities:
+@Kovri realisiert verschiedene Arten von Verschlüsselung idurchn *mindestens* vier erforderliche Funktionen:
```

Comment:
```suggestion
@Kovri realisiert verschiedene Arten von Verschlüsselung durch *mindestens* vier erforderliche Funktionen:
```

Reply:
Not sure if "capacities" is correctly translated to "Funktionen" here.. I read that sentence more like "in at least 4 essential levels" - with "Reseed" being the first level, upon which Garlic-Routing is built (2nd level), upon which "Tunnel encryption" is built (3rd level), and so on.. but I could be wrong, don't know much about kovri.

Reply:
> I could be wrong, don't know much about kovri
Same for me. Maybe @erciccione could help out here.

Reply:
> I read that sentence more like "in at least 4 essential levels"

More like "in at least 4 different ways" as far as i understood. After reading these comments i realized this section needs to be removed reguardless (#1311), so don't worry too much about it.

Reply:
Ok then, to simplify matters I leave it as it is?

Reply:
Yeah, it's not a big deal.